plugins/webp-uploads/helper.php

```diff
@@ -144,7 +145,7 @@ function webp_uploads_generate_additional_image_source( int $attachment_id, stri
 		return new WP_Error( 'image_mime_type_not_supported', __( 'The provided mime type is not supported.', 'webp-uploads' ) );
 	}
 
-	$image_path = wp_get_original_image_path( $attachment_id );
+	$image_path = wp_attachment_is_image( $attachment_id ) ? wp_get_original_image_path( $attachment_id ) : get_attached_file( $attachment_id );
```

Comment:
Let's still avoid the ternary for readability and to ensure we have code coverage for both conditions:

```suggestion
	if ( wp_attachment_is_image( $attachment_id ) ) {
		$image_path = wp_get_original_image_path( $attachment_id );
	} else {
		$image_path = get_attached_file( $attachment_id );
	}
```